subworkflows/local/align_cellrangermulti.nf

```diff
@@ -97,7 +97,12 @@ workflow CELLRANGER_MULTI_ALIGN {
                     lines = [ 'sample_id,cmo_ids,description' ]
                     meta[0].each{ lines = lines + [ "$it.multiplexed_sample_id,$it.cmo_ids,$it.description" ] }
                     cmos = file( "${workflow.workDir}/${sample}_cmo_samplesheet.csv" )
```

Comment:
I'm not sure this is a good idea.

Does this work on an AWS batch setup?
there could be clashes if multiple workflows run with the same work dir
the file could be leftover from previous pipeline runs
Maybe this needs to become its own process that has its own work dir, or at least write to a unique filename.

Reply:
First. I wanted to created it as a `.collectFile()` function so that the unique "filename" status would be achieved and allowing `caching`. However I could not. Maybe, indeed, having it as a process would allow it.

Reply:
I will check if I can still make it with `.collectFile()`. Otherwise, the module solution might be the easiest.

Reply:
Will do it as a module.

Reply:
It as done as suggested as a module and it is discussed here: https://github.com/nf-core/scrnaseq/pull/276#discussion_r1562417688

Closing this one.

Reply:
Just stumbled across this somewhere else: I think `collecFile` would be a way to write this all in nextflow/groovy that should be safe to use across different setups: https://www.nextflow.io/docs/latest/operator.html#collectfile

(no changes needed here, the module is fine -- just for future reference)